### DISCUSSION TITLE
[Run issue]:IRTCummulative example returns an empty file

### DISCUSSION THREAD

**Discussion by alex-phy** [topas-nbio/TOPAS-nBio#57] [Run-Problem]
### Contact Details

guimaraes.alexf@gmail.com

### What was the issue?

I am having difficulty running the IRTCummulative example because the output .phsp file is empty. I have been analyzing the code and couldn't find the scorer file mentioned (s:Sc/gvalue/Quantity = "TsIRTInterPulse") either in my installation or in the Github repository.

### TOPAS-nBio category

Scoring

### Version

3.0

### What Operative System (OS) are you running on?

Linux (Debian/Ubuntu)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi,

Just checked on my end and I can't reproduce this issue - the .phsp file produced after running the example is not empty.

The source code for the scorer is located in scorers/TsScoreWithIRTCummulative.cc. My recommendation would be to re-install TOPAS-nBio as something might've gone wrong in installation.

All the best,
Thongchai

**Reply by alex-phy** [reply]
Hi, 

I reinstalled TOPAS-nBio and ran the TsIRTCummulative example with protons, and it worked. However, with electrons, the .phsp files return empty.

Best, 
Alex

**Reply by tmasilela** [reply]
Hi Alex,

If you just changed the particle type in the example then the electrons are too energetic and will be unlikely to interact in the small volume. If you change the energy to 50 keV for example then you shouldn't see any issues. Hope that helps!

All the best,
Thongchai

**Comment by Yasminjansen** [comment]
Hi,

yasminjaraujo@gmail.com

I'm trying to install TOPAS-nBio, I followed the step-by-step instructions at https://github.com/OpenTOPAS/OpenTOPAS/blob/main/OpenTOPAS_quickStart_WSL.md, but I keep getting an error message on step 8. I'd like to know how you were able to install TOPAS.

Best,
Yasmin
![WhatsApp Image 2025-02-16 at 20 29 31 (1)](https://github.com/user-attachments/assets/0ba3316b-66d4-4e87-bd7c-bffd75dcdec9)

**Comment by tmasilela** [comment]
Hi Yasmin,

I will respond here but I would encourage you to make a new discussion to get help with installation issues.

Based on what you provided your system isn't finding some Geant4 files, which is likely due to the Geant4 export directory being incorrect. Please ensure that the path you provided to Geant4_DIR= ... is correct.

Hope that helps, and all the best,
Thongchai
